tests/win_sec_attr.c

```diff
@@ -0,0 +1,96 @@
+/*
+    Copyright (c) 2012 Martin Sustrik  All rights reserved.
```

Comment:
Did you copy Martin's code?  I'm surprised to see *only* his copyright here, rather than the actual author's.